backend/domain/services/conversation_manager.py

```
@@ -0,0 +1,62 @@
+        user_id: str,
+        model_name: str,
+        messages: list[Message],
+        thread_id: str | None = None,
+    ) -> ConversationContext:
+        """会話コンテキストを作成します。
+
+        Args:
+            user_id: ユーザーID
+            model_name: 使用するモデル名
+            messages: 初期メッセージリスト
+            thread_id: スレッドID（新規の場合はNone）
```

Ruff (0.14.11)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)